contracts/WildcardSteward_v2.sol

```diff
@@ -269,6 +283,15 @@ contract WildcardSteward_v2 is Initializable {
 
             totalPatronTokenGenerationRate[currentOwner] = totalPatronTokenGenerationRate[currentOwner]
                 .add(11574074074074);
+
+            address tokenBenefactor = benefactors[tokenId];
+
+            benefactorTotalTokenNumerator[tokenBenefactor] = benefactorTotalTokenNumerator[tokenBenefactor]
+                .add(price[tokenId].mul(patronageNumerator[tokenId]));
+
+            if (benefactorLastTimeCollected[tokenBenefactor] == 0) {
```

Comment:
This is a brand new variable, think its going to be ==0 in every case? Is there any other reason for this if check?

Reply:
It gets set at the beginning of every function due to the `_updateBenefactorBalance` function. So it shouldn't be an issue I think. But there might be edge cases to look for here.